virt/virt-4-10-release-notes.adoc

```diff
@@ -284,3 +284,9 @@ $ oc annotate --overwrite -n openshift-cnv hco kubevirt-hyperconverged 'networka
 
 * The web console does not display virtual machine templates that are deployed to a custom namespace. Only templates deployed to the default namespace display in the web console. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2054650[*BZ#2054650*])
 ** As a workaround, avoid deploying templates to a custom namespace.
+
+* If a VMI has the `spec.evictionStrategy` field set to `LiveMigrate`, then you cannot migrate or evict the VMI or upgrade a cluster. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2073880[*BZ#2073880*])
```

Comment:
I'm just making sure...

This release note applies ONLY when running Single Node OpenShift. Should that clearly be stated here or is the entire section about SNO?